src/popup/pages/AccountDetailsNames.vue

```diff
@@ -1,23 +1,26 @@
 <template>
   <div class="account-details-names">
-    <transition
-      name="fade-transition"
-      mode="out-in"
+    <RouterView
+      v-if="isOnline"
+      v-slot="{ Component }"
     >
-      <RouterView
-        v-if="isOnline"
-      />
-      <MessageOffline
-        v-else
-        class="offline-message"
-        :text="$t('modals.accountDetails.namesNotAvailable')"
-      />
-    </transition>
+      <transition
+        name="fade-transition"
+        mode="out-in"
+      >
+        <component :is="Component" />
```

Comment:
Please use PascalCase for the component name.

Reply:
https://github.com/aeternity/superhero-wallet/pull/2135/files#diff-a8a9f13cb04f4ba37a0d111cada9b87e272f8f684425f47dff7be9bf8ea94eafR39 for example

And others
![image](https://github.com/aeternity/superhero-wallet/assets/9008074/39613cd6-b0b4-4b6c-9884-f18da0ea223c)